scripts/generate_readme.go

```diff
@@ -0,0 +1,106 @@
tion are\nprovided, and you can query any data source you want by implementing a\nfew interfaces.\n\n**go-mysql-server** also provides a server implementation compatible\nwith the MySQL wire protocol. That means it is compatible with MySQL\nODBC, JDBC, or the default MySQL client shell interface.\n\n[Dolt](https://www.doltdb.com), a SQL database with Git-style \nversioning, is the main database implementation of this package. \nCheck out that project for reference implementations. Or, hop into the Dolt discord [here](https://discord.com/invite/RFwfYpu)\nif you want to talk to the core developers behind GMS.\n\n## Scope of this project\n\nThese are the goals of **go-mysql-server**:\n\n- Be a generic extensible SQL engine that performs queries on your\n  data sources.\n- Provide a simple database implementation suitable for use in tests.\n- Define interfaces you can implement to query your own data sources.\n- Provide a runnable server speaking the MySQL wire protocol,\n  connected to data sources of your choice.\n- Optimize query plans.\n- Allow implementors to add their own analysis steps and\n  optimizations.\n- Support indexed lookups and joins on data tables that support them.\n- Support external index driver implementations such as pilosa.\n- With few caveats and using a full database implementation, be a\n  drop-in MySQL database replacement.\n\nNon-goals of **go-mysql-server**:\n\n- Be an application/server you can use directly.\n- Provide any kind of backend implementation (other than the `memory`\n  one used for testing) such as json, csv, yaml. That's for clients to\n  implement and use.\n\nWhat's the use case of **go-mysql-server**?\n\n**go-mysql-server** has two primary uses case:\n\n1. Stand-in for MySQL in a golang test environment, using the built-in\n   `memory` database implementation.\n\n2. Providing access to aribtrary data sources with SQL queries by\n   implementing a handful of interfaces. The most complete real-world\n   implementation is [Dolt](https://github.com/dolthub/dolt).\n\n## Installation\n\nThe import path for the package is `github.com/dolthub/go-mysql-server`.\n\nTo install it, run:\n\n```\ngo get github.com/dolthub/go-mysql-server\n```\n\n## Go Documentation\n\n* [go-mysql-server godoc](https://godoc.org/github.com/dolthub/go-mysql-server)\n\n## SQL syntax\n\nThe goal of **go-mysql-server** is to support 100% of the statements\nthat MySQL does. We are continuously adding more functionality to the\nengine, but not everything is supported yet. To see what is currently\nincluded check the [SUPPORTED](./SUPPORTED.md) file.\n\n## Third-party clients\n\nWe support and actively test against certain third-party clients to\nensure compatibility between them and go-mysql-server. You can check\nout the list of supported third party clients in the\n[SUPPORTED_CLIENTS](./SUPPORTED_CLIENTS.md) file along with some\nexamples on how to connect to go-mysql-server using them.\n## Available functions\n\n<!-- BEGIN FUNCTIONS -->\n"
```

Comment:
Rather than this, do:

1) Read the README and echo it until you see <!-- BEGIN FUNCTIONS -->
2) Output functions
3) Skip past README until you see <!--END Functions -->
4) Echo the rest